tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-interactive-inference-dashboard.html

```diff
@@ -2829,6 +2849,70 @@ <h2>Create a distance feature</h2>
         this.selectedExampleAndInference =
           this.selected.length > 0
           ? this.examplesAndInferences[this.selected[0]] : null;
+        requestAnimationFrame(()=> this.setAttributions());
```

Comment:
for legibility, can you put one whitespace before `=>`? thanks!
also, often, if you start a RAF, it is a good practice to cancel it on unmount or `detached`. Please consider cancelling the RAF.

Reply:
done